convolution.py

```diff
@@ -0,0 +1,30 @@
+import numpy as np
+import scipy.stats
+from scipy.stats import gamma
+from stimuli import events2neural
+
+def hrf(times):
+    peak_values = gamma.pdf(times,6)
+    undershoot_values = gamma.pdf(times, 12)
+    values = peak_values-0.35*undershoot_values
+    
+    return values/np.max(values)*0.6
```

Comment:
Based on lecture, I remembered it is just a way to combine different gamma distributions  in order to form a distribution has peak and lowest value. We can change these parameter to see how is perform, but prof did go through math theory behind